Gordon360/Controllers/VersionController.cs

```diff
@@ -16,14 +17,9 @@ public class VersionController : ControllerBase
         [Route("")]
         public ActionResult<string> Get()
         {
-            string gitVersion = string.Empty;
-            using (Stream stream = Assembly.GetExecutingAssembly()
-                    .GetManifestResourceStream("Gordon360." + "version.txt"))
-            using (StreamReader reader = new StreamReader(stream))
-            {
-                gitVersion = reader.ReadLine();
-            }
-            return Ok(gitVersion);
+            var asm = typeof(VersionController).Assembly;
+            var attrs = asm.GetCustomAttributes<AssemblyMetadataAttribute>();
+            return $"Git Hash: {attrs.FirstOrDefault(a => a.Key == "SourceRevisionId")?.Value}";
```

Comment:
Can make the returned string be JSON with keys "commit" and "date"?  The front end can then format the output more easily.